transport/src/test/java/io/netty/channel/pool/FixedChannelPoolTest.java

```diff
@@ -318,6 +319,25 @@ public void run() {
         sc.close().syncUninterruptibly();
     }
 
+    @Test
+    public void testAcquireAfterClosePool() throws Exception {
```

Comment:
@lutovich remove ?

Reply:
Should the test be removed or the exception message assertion?

Reply:
I would say the test and if you want to do another PR to add more tests just do it there. The test has not really anything to do with the change of the message.